tests/lax_numpy_test.py

```diff
@@ -256,6 +256,7 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
     op_record("true_divide", 2, all_dtypes, all_shapes, jtu.rand_nonzero,
               ["rev"], inexact=True),
     op_record("diff", 1, number_dtypes, nonzerodim_shapes, jtu.rand_default, ["rev"]),
+    op_record("ediff1d", 3, [jnp.int64], all_shapes, jtu.rand_default, []),
```

Comment:
This is not possible, as some of the combinations of dtypes are not possible to cast into each other. I could add some tests with other dtypes.

Reply:
Ah, okay. In that case, let's still change this to use `onp` instead of `jnp` for consistency with the others. Also, let's use `onp.int32` instead of `int64`, so that behavior won't change with the global `jax_enable_x64` flag.

Reply:
Okay, that is now incorporated!